orchagent/switchorch.cpp

```diff
@@ -49,6 +58,20 @@ SwitchOrch::SwitchOrch(DBConnector *db, vector<TableConnector>& connectors, Tabl
     querySwitchTpidCapability();
     auto executorT = new ExecutableTimer(m_sensorsPollerTimer, this, "ASIC_SENSORS_POLL_TIMER");
     Orch::addExecutor(executorT);
+
+    sai_attribute_t attrs[2];
```

Comment:
Please use vector instead to avoid hard coded size.
Example: https://github.com/Azure/sonic-swss/blob/master/orchagent/portsorch.cpp#L474